packages/components/src/LightBox/LightBox.module.css

```
@@ -15,71 +15,125 @@
   }
 }
 
+.backgroundImage {
+  position: absolute;
+  inset: 0;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  opacity: 0.6;
+  filter: saturate(67%);
+}
+
 .next,
 .prev {
-  display: flex;
   position: absolute;
-  top: 50%;
-  z-index: var(--elevation-base);
-  transform: translateY(-50%);
+  top: 43%;
+  z-index: calc(var(--elevation-base) + 1);
 }
 
 .next {
-  right: var(--space-base);
+  right: var(--space-large);
+  @media (--medium-screens-and-up) {
+    right: var(--space-larger);
+  }
 }
 
 .prev {
-  left: var(--space-base);
+  left: var(--space-large);
+  @media (--medium-screens-and-up) {
+    left: var(--space-larger);
+  }
+}
+
+.buttonHidden {
+  opacity: 0;
+  transition: opacity var(--timing-base) ease-in;
+
+  &:hover {
```

```suggestion
  &:hover,
  &:focus-within {
```

so that if you are using your keyboard on these buttons directly, it stays showing too

I know you can use your arrow keys to do the same thing, but since they're focusable, we might as well make them work